### PR TITLE
Add baseline option for analyze command

### PR DESCRIPTION
**Pull request by yongyan-gh**

### CODE REVIEW

File: src/Sarif.Driver/Sdk/PluginDriverCommand.cs [hunk 1]
```diff
@@ -30,5 +36,67 @@ public IEnumerable<Assembly> RetrievePluginAssemblies(IEnumerable<Assembly> defa
...
+                Formatting = options.PrettyPrint || (!options.PrettyPrint && !options.Minify) ?
+                                    Formatting.Indented :
+                                    Formatting.None
+            };
+
+            SarifLog baselineFile;
+            using (var reader = new JsonTextReader(new StreamReader(fileSystem.FileOpenRead(options.BaselineSarifFile))))
+            {
+                baselineFile = serializer.Deserialize<SarifLog>(reader);
+            }
```
Comment (eddynaka) [inline src/Sarif.Driver/Sdk/PluginDriverCommand.cs:63]: you can replace this by:
```
baselineFile = SarifLog.Load(optons.BaselineSarifFile)
```

File: src/Sarif.Driver/Sdk/PluginDriverCommand.cs [hunk 2]
```diff
@@ -30,5 +36,67 @@ public IEnumerable<Assembly> RetrievePluginAssemblies(IEnumerable<Assembly> defa
...
+
+            SarifLog currentSarifLog;
+            using (var reader = new JsonTextReader(new StreamReader(fileSystem.FileOpenRead(options.OutputFilePath))))
+            {
+                currentSarifLog = serializer.Deserialize<SarifLog>(reader);
+            }
```
Comment (eddynaka) [inline src/Sarif.Driver/Sdk/PluginDriverCommand.cs:69]: use this:
```
currentSarifLog = SarifLog.Load(xyz)
```

**Review by michaelcfanning — APPROVED** [review]
:shipit:

File: src/Sarif.Driver/Sdk/PluginDriverCommand.cs (hunk 1 above)
Comment (yongyan-gh) [reply src/Sarif.Driver/Sdk/PluginDriverCommand.cs]: Can I inject a mock IFileSystem to SarifLog for test cases?

**Pull request merged**